frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts

```diff
@@ -0,0 +1,75 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { dashboardIsLoaded } from '@console/shared/src/test-views/dashboard-shared.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  mainHealtGreenSvg,
+  mainHealthCardStatus,
+  mainHealtRedSvg,
+  mainHealtYellowSvg,
+  noOutChange,
+  seeAllClose,
+  seeAllLink,
+  seeAllStorageGreenSvg,
+  seeAllStorageRedSvg,
+  seeAllStorageSection,
+  seeAllStorageYellowSvg,
+} from '../../views/ocp-dashboard-card-healthcheck.view';
+
+describe('Check health data on on main OCP dashboard and see all link.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/dashboards`);
+    await dashboardIsLoaded();
+  });
+
+  it('Check main dashboard health is green', async () => {
+    expect(mainHealtGreenSvg.isPresent()).toBe(true);
+    expect(mainHealthCardStatus.getText()).toContain('healthy');
```

Comment:
Its better to create an constant for `healthy` as its used at multiple places.

Reply:
Created an enum for all the ocp text statuses:
https://github.com/openshift/console/blob/2a4df9e82d79a40e4c69f7541226b98979d22631/frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts#L45-L49